### PR TITLE
 ReduceSum

### PR DESCRIPTION
**Pull request by Deliadd**

### CODE REVIEW

File: pyzoo/test/zoo/pipeline/onnx/test_model_loading.py [hunk 1]
```diff
@@ -452,3 +452,16 @@ def forward(self, x):
         pytorch_model = MnistNet()
         pytorch_model.train(mode=False)
         self.compare_with_pytorch(pytorch_model, [(1, 1, 28, 28)])
+
+    def test_onnx_reducesum(self):
+        class ReduceSum(torch.nn.Module):
+            def forward(self, x):
+                for i in range(len(x)):
+                    total = torch.sum(x[i])
```
Comment (zhichao-li) [inline pyzoo/test/zoo/pipeline/onnx/test_model_loading.py]: `torch.sum(x, axis, keep_dim)`?

**Comment by Deliadd** [comment]
Jenkins passed. ID is 1448.
